chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,98 +1,508 @@
 {
 	"info": {
-		"_postman_id": "21c26d32-3ea4-4d99-ac3d-c4c557baa398",
+		"_postman_id": "ec8c7252-3caa-48d3-8060-d247a94d6ea3",
 		"name": "developer-controller",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
 		{
-			"name": "GET /developers -invalid API Key header",
-			"event": [
+			"name": "GET : Developers: data driven tests",
+			"item": [
 				{
-					"listen": "test",
-					"script": {
-						"id": "89af904b-9c4d-4c17-9fd1-83ff5d945beb",
-						"exec": [
-							"pm.test(\"Invalid api key header should return 401 status\", function () {\r",
-							"    pm.response.to.have.status(401);\r",
-							"});"
+					"name": "GET /hierarchy - valid Deverloper Ids",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "1b156faa-475d-482e-b573-f0d184a409fa",
+								"exec": [
+									"const developerId= pm.iterationData.get(\"developerId\");\r",
+									"pm.environment.set(\"developerId\",\"developerId\");"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "5f35de1e-0aa3-4d75-93d8-b1c451e372bf",
+								"exec": [
+									"pm.test(\"Valid api key header should return 200 status and response is not  null\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData.response).not.eq(null);\r",
+									"});\r",
+									"pm.test(\"Developer Names are correct as per the id passed in\", function () {\r",
+									"    var actualBody= pm.response.json();\r",
+									"    var expected= pm.iterationData.get(\"name\");\r",
+									"    pm.expect(actualBody.name).to.equal(expected);\r",
+									"    var expected= pm.iterationData.get(\"developerCode\");\r",
```

Comment:
Does this line redefine the `expected` variable? It shouldn't do that

Reply:
This is done